core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/jndi/JndiDirectoryDAO.scala

```diff
@@ -192,9 +192,25 @@ trait JndiDirectoryDAO extends DirectorySubjectNameSupport with JndiSupport {
     }
 
     def load(groupRefs: TraversableOnce[RawlsGroupRef]): ReadWriteAction[Seq[RawlsGroup]] = withContext { ctx =>
+      val (policies, workbenchGroups) = groupRefs.toSeq.partition(ref => ref.groupName.value.matches(policyGroupNamePattern.regex))
+
+      for {
+        loadedPolicies <- loadSequentially(policies)
```

Comment:
messed with filtering for batch loading policies for a bit without success

may be worth looking into in the future, but the amount of policies that should be loaded individually should be _relatively_ low even for somebody with a lot of workspaces